src/test/ui/existential_types/generic_duplicate_param_use4.rs

```diff
@@ -0,0 +1,15 @@
+#![feature(existential_type)]
+
+fn main() {}
+
+// test that unused generic parameters are ok
+existential type Two<T, U>: 'static;
+
+fn one<T: 'static>(t: T) -> Two<T, T> {
+    t
+}
+
+fn three<T: 'static, U: 'static>(_: T, u: U) -> Two<T, U> {
```

Comment:
Oh, nvm-- it looks like the test I asked for above is basically the same as this one. I think it's a bug that this function not be allowed but the example with `one`, `two` is-- there's no reason that the order of the type parameters in the `existential type` should affect the result.

Reply:
Yea... I think my implementation of that algorithm may be very naive (which was ok as long as unused parameters were not allowed). I'll implement this properly